src/domains/printable.ml

```diff
@@ -433,6 +453,8 @@ struct
     BatPrintf.fprintf f "<value>\n<map>\n";
     loop 0 xs;
     BatPrintf.fprintf f "</map>\n</value>\n"
+
+  let to_yojson xs = `List (BatList.map Base.to_yojson xs)
```

Comment:
Is this already not the default derived implementation for lists?

Reply:
You're right!